documentation/book/proc-scaling-down-a-kafka-cluster.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// ???
+
+[id='proc-scaling-down-a-kafka-cluster-{context}']
+
+= Scaling down a Kafka cluster
+
+This procedure describes how to decrease the number of brokers in a Kafka cluster.
+
+.Prerequisites
+
+* An existing Kafka cluster.
+* A _reassignment JSON file_, `_reassignment.json_`, of how partitions should be reassigned to brokers in the cluster once the broker(s) in the highest numbered `Pod(s)` have been removed.
```

Comment:
A _reassignment JSON file_ named `_reassignment.json_` describing how partitions should be reassigned to brokers in the cluster after the broker(s) in the highest numbered `Pod(s)` have been removed.